docs/release-notes/RELEASE-NOTES-6.0.0-rc4.md

```diff
@@ -0,0 +1,35 @@
+# About this release
+
+[This](https://github.com/aeternity/aeternity/releases/tag/v6.0.0-rc4) is the fourth Iris release candidate.
+
+It:
+
+* Fixed the default value of the gas limit for the `external` dry-run
+  endpoint. It used to have a default of 1 000 000 gas per call and now it is
+  6 000 000 per call.
+
+* Set height for `iris` hard fork to happen on block 441444, on 10 June 2021,
+  around 9:11am UTC
+
+Please test the **iris** protocol by activating it in the configuration file as below:
```

Comment:
I think we can drop this already ? Testnet switched already, there is no reason one to switch their mainnet node manually?